PlaybackLab/ControlsLockScreenSampleApp/ControlsLockScreenSampleApp/Views/ListTableViewController.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  ListTableViewController.swift
+//  ControlsLockScreenSampleApp
+//
+//  Created by Carlos Ceja on 10/18/17.
+//  Copyright © 2017 Ooyala. All rights reserved.
+//
+
+import UIKit
+
+class ListTableViewController: UITableViewController {
+  
+  private let options: [PlayerSelectionOption] = OptionDataSource.options()
+  
+  private var cellIdentifier: String = "cellOption"
+  
+  override func viewDidLoad() {
+    super.viewDidLoad()
+    tableView.tableFooterView = UIView()
+    tableView.bounces = false
+  }
+  
+  override var supportedInterfaceOrientations: UIInterfaceOrientationMask {
+    return .portrait
+  }
+  
+  override var shouldAutorotate: Bool {
+    return false
+  }
+  
+  // MARK: - Table view data source
+  
+  override func numberOfSections(in tableView: UITableView) -> Int {
```

Comment:
You can remove this method since it defaults to 1

Reply:
Done, I removed it.